docs-src/references/events.md

```diff
@@ -31,7 +31,7 @@ It indicates that the Cluster received a request to spawn the Workflow Execution
 | continued_failure                  | Serialized result of a failure.                                                                                                                          |
 | last_completion_result             | Information from the previously completed [Task](/concepts/what-is-a-task), if applicable.                                                               |
 | original_execution_run_id          | The [Run Id](/concepts/what-is-a-run-id) of the original Workflow started.                                                                               |
-| identity                           | Id of the [Worker](/concepts/what-is-a-worker) that requested the start of this Workflow.                                                                |
+| identity                           | Id of the [Client](/clusters#temporal-client-connections) that requested the start of this Workflow.                                                     |
```

Comment:
```suggestion
| identity                           | Id of the [Client](/clusters#temporal-client-connections) or parent Workflow [Worker](/concepts/what-is-a-worker) that requested the start of this Workflow.                                                     |
```

Reply:
And you should be able to do `/concepts/what-is-a-temporal-cluster#temporal-client-connections` and get a link preview for that link.